domino.go

```diff
@@ -668,6 +772,211 @@ func (o *putOutput) Result(item interface{}) (err error) {
 	return
 }
 
+/***************************************************************************************/
+/************************************** TransactWriteItems *********************************/
+/***************************************************************************************/
+type transactWriteItemsInput struct {
+	batches          []*dynamodb.TransactWriteItemsInput
+	table            DynamoTable
+	delayedFunctions []func() error
+	requestToken     string
+}
+type transactWriteItemsOutput struct {
+	*dynamoResult
+	results []*dynamodb.TransactWriteItemsOutput
+}
+
+/*TransactWriteItems represents dynamo batch write item call*/
+func (table DynamoTable) TransactWriteItems() *transactWriteItemsInput {
+	r := transactWriteItemsInput{
+		batches: nil,
+		table:   table,
+	}
+	return &r
+}
+
+func (d *transactWriteItemsInput) WithClientRequestToken(token string) *transactWriteItemsInput {
+	d.requestToken = token
+	return d
+}
+
+func (d *transactWriteItemsInput) writeItem(item interface{}, f func(DynamoDBValue) *dynamodb.TransactWriteItem) *transactWriteItemsInput {
+
+	delayed := func() error {
+		if d.batches == nil {
+			d.batches = []*dynamodb.TransactWriteItemsInput{}
+		}
+		batchLen := len(d.batches)
+
+		var batch *dynamodb.TransactWriteItemsInput
+		// Create a new batch if there are none, or the last batch >= 10
+		if batchLen == 0 || len(d.batches[batchLen-1].TransactItems) >= 10 {
```

Comment:
it seems like we use this `10` number in a lot of places - could we possibly make this into a packge-level constant